hvplot/plotting/scatter_matrix.py

```diff
@@ -43,7 +46,22 @@ def scatter_matrix(data, c=None, chart='scatter', diagonal='hist',
         Default is `Category10 <https://github.com/d3/d3-3.x-api-reference/blob/master/Ordinal-Scales.md#category10>`.
     diagonal_kwds/hist_kwds/density_kwds: dict, optional
         Keyword options for the diagonal plots
-    kwds: Keyword options for the off-diagonal plots, optional
+    datashade (default=False):
+        Whether to apply rasterization and shading using datashader
+        library returning an RGB object
+    rasterize (default=False):
+        Whether to apply rasterization using the datashader library
+        returning an aggregated Image
```

Comment:
```suggestion
    datashade (default=False):
        Whether to apply rasterization and shading (colormapping) using
        the Datashader library, returning an RGB object instead of 
        individual points
    rasterize (default=False):
        Whether to apply rasterization using the Datashader library,
        returning an aggregated Image (to be colormapped by the 
        plotting backend) instead of individual points
```